dev/sg/main.go

```diff
@@ -116,12 +116,29 @@ func checkSgVersion() {
 	}
 
 	out = strings.TrimSpace(out)
-	if out != "" {
+	if out == "" {
+		// No newer commits found. sg is up to date.
+		return nil
+	}
+
+	if *skipAutoUpdatesFlag {
 		stdout.Out.WriteLine(output.Linef("", output.StyleSearchMatch, "------------------------------------------------------------------------------"))
 		stdout.Out.WriteLine(output.Linef("", output.StyleSearchMatch, "       HEY! New version of sg available. Run 'sg update' to install it.       "))
 		stdout.Out.WriteLine(output.Linef("", output.StyleSearchMatch, "             To see what's new, run 'sg version changelog -next'.             "))
 		stdout.Out.WriteLine(output.Linef("", output.StyleSearchMatch, "------------------------------------------------------------------------------"))
+	} else {
```

Comment:
Since the other branch returns, this might be clearer:

```suggestion
		return nil
	}
```